README.md

```diff
@@ -349,8 +349,19 @@ $ minikube start --docker-env HTTP_PROXY=http://$YOURPROXY:PORT \
                  --docker-env HTTPS_PROXY=https://$YOURPROXY:PORT
 ```
 
+## Minikube Environment Variables
+Minikube supports passing environment variables instead of flags for some flag values.  For example the `minikube start --iso-url="$ISO_URL"` flag can also be set by setting the MINIKUBE_ISO_URL="$ISO_URL" environment variable.
 
-## Known Issues
+Some features can only be accessed by environment variables, here is a list of these features:
+MINIKUBE_HOME - (string) sets the path for the .minikube directory that minikube uses for state/configuration
+MINIKUBE_WANTUPDATENOTIFICATION - (bool) sets whether the user wants an update notification for new minikube versions
+MINIKUBE_REMINDERWAITPERIODINHOURS - (int) sets the number of hours to check for an update notification
+MINIKUBE_WANTREPORTERROR - (bool) sets whether the user wants to send anonymous errors reports to help improve minikube
+MINIKUBE_WANTREPORTERRORPROMPT - (bool) sets whether the user wants to be prompted on an error that they can report them to help improve minikube
+MINIKUBE_WANTKUBECTLDOWNLOADMSG - (bool) sets whether minikube should tell a user that `kubectl` cannot be found on there path
+

```

Comment:
I would also add that anything under `minikube config view` can be controlled by a environment variable with the prefix `MINIKUBE_`

You might also want to include the `ENABLE_TRACING` var (I forget the exact name)